src/Controller/CheckListController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+
+namespace App\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\Routing\Annotation\Route;
+
+class CheckListController
+{
+    /**
+     * @Route ("/checklist", name="checklist")
+     * @Template("checklist.html.twig")
```

Comment:
Your template annotations should not have a name. Let it autoload, like this
`@Template()`